docs/scripts.rst

```diff
@@ -365,6 +375,44 @@ These values are stored in the ``fields`` array of the ``schema`` dict of the JS
   year, int
   ...
 
+Consider the example of schema of table given below:
```

Comment:
I think this section (lines 378-415) might be a little redundant with the lines that are around it. It might be better to use the Mammal schema already provided and explain the fields with that one, along with a list of all possible field types like the one henry provided: 

```
pk-auto: Auto generated primary key starting from 1
pk-[char,int,double]: primary key with data type
char: strings
int: integers
double:floats/decimals
ct-[int,double,char]:Cross tab data
```